demos/applications/icm20948.cpp

```diff
@@ -13,10 +13,25 @@
 // limitations under the License.
 
 #include "../hardware_map.hpp"
+#include <cmath>
 #include <libhal-icm/icm20948.hpp>
 #include <libhal-util/serial.hpp>
 #include <libhal-util/steady_clock.hpp>
```

Comment:
```suggestion
#include <cmath>

#include <libhal-icm/icm20948.hpp>
#include <libhal-util/serial.hpp>
#include <libhal-util/steady_clock.hpp>

#include "../hardware_map.hpp"
```

I need to put the include order into the libhal documentation, but it should follow the Google style of header inclusion. We do not follow all of the C++ Google standards, especially since we will be moving to exceptions soon and we use things like std::uint32_t, but we will be following much of it. 

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes